src/main.js

```diff
@@ -4,29 +4,38 @@ import Vue from 'vue'
 import App from './App'
 import VueGtag from 'vue-gtag'
 import router from './router'
+import * as Sentry from '@sentry/browser';
+import {Vue as VueIntegration} from '@sentry/integrations';
+
+Sentry.init({
+    dsn: 'https://49674290a25345fa80a1191010a6cbbb@o422063.ingest.sentry.io/5343662',
+    integrations: [new VueIntegration({Vue, attachProps: true})],
+});
 
 var VueCookie = require('vue-cookie');
 
 Vue.use(VueGtag, {
-  config: {
-    id: 'UA-167215120-1'
-  }
+    config: {
+        id: 'UA-167215120-1'
```

Comment:
Is there any reason to add additional indentation here? Also applies to Line 18,20,37,38,39,40.

Reply:
I didn't mean to indent it, but auto reformatting function makes some additional indent here. I'll recheck about it with next commit.